shared/postgres/scripts/insert_data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+cleanup() {
+  echo "Interrupted. Exiting..."
+  exit
+}
+
+trap cleanup SIGINT
+
+# Number of records per batch
+batch_size=80000
+
+# Number of batches
+batches=200
```

Comment:
Given that this is a common script, maybe we should rather pass it in as an argument, so you can control it in the benchmark.